tests/crameri_benchmark_1_gmg.prm

```diff
@@ -0,0 +1,59 @@
+# Can check this benchmark in gnuplot by opening it and running:
```

Comment:
For all prms added, change the description at the top to indicate that this is to test gmg+free surface.